examples/user_interfaces/embedding_in_wx3_sgskip.py

```diff
@@ -56,8 +56,8 @@ def __init__(self, parent):
     def init_plot_data(self):
         ax = self.fig.add_subplot()
 
-        x = np.arange(120.0) * 2 * np.pi / 60.0
-        y = np.arange(100.0) * 2 * np.pi / 50.0
+        x = np.arange(120.0) * 2 * np.pi / 60
+        y = np.arange(100.0) * 2 * np.pi / 50
```

Comment:
```suggestion
        x = np.arange(120) * 2 * np.pi / 60
        y = np.arange(100) * 2 * np.pi / 50
```
Does this change things?

Reply:
Yes, it makes it much faster. And the result is still in `np.float`. Thank you, I changed it.